src/main/java/com/google/swarm/tokenization/beam/DLPDeidentifyText.java

```diff
@@ -213,6 +220,10 @@ public PCollection<KV<String, DeidentifyContentResponse>> expand(
     private final PCollectionView<Map<String, List<String>>> headerColumns;
     private transient DeidentifyContentRequest.Builder requestBuilder;
     private transient DlpServiceClient dlpServiceClient;
+    private static final FluentBackoff BUNDLE_WRITE_BACKOFF =
+            FluentBackoff.DEFAULT
+                    .withMaxRetries(10)
+                    .withInitialBackoff(Duration.standardSeconds(5));
```

Comment:
Same here. Please us a constant for default and expose this as an optional pipeline argument. This will allow callers to customise behaviour. The constant should probably be in Util.
Initialisation will probably move to constructor.